lms/services/lti_grading/_v13.py

```diff
@@ -66,7 +66,7 @@ def record_result(self, grading_id, score=None, pre_record_hook=None):
                 scopes=self.LTIA_SCOPES,
                 json=payload,
                 headers={"Content-Type": "application/vnd.ims.lis.v1.score+json"},
-            ).json()
```

Comment:
This is used in:

https://github.com/hypothesis/lms/blob/fac2eaf8cd6167abae6bf1db9da38b8fea860778/lms/views/api/grading.py#L30

https://github.com/hypothesis/lms/blob/fac2eaf8cd6167abae6bf1db9da38b8fea860778/lms/views/api/grading.py#L86

which don't read the value back